source/jormungandr/jormungandr/scenarios/ridesharing/ouestgo.py

```diff
@@ -50,6 +56,10 @@
 }
 
 
+def format_str_datetime(str_date, str_time):
+    return '{}T{}'.format(str_date.replace('-', ''), str_time.replace(':', ''))
```

Comment:
Manually parsing and constructing date and time seems brittle. Relying on the standard library seems a good option here

For example, you should be able to get a valid `datetime` with the following.
```python
import datetime

date = datetime.date.fromisoformat(str_date)
time = datetime.time.fromisoformat(str_time)
dt = datetime.datetime.combine(date, time, timezoneinfo)
```